activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+*   Introduce ApplicationRecord, an Active Record layer supertype.
+
+    Plugin authors can now distribute Active Record extensions as
```

Comment:
Other motivation, and I believe the main one, is to isolate engine models from application models. Now engine are free to use Active Record extensions, change configurations, etc without reflecting in how the application works. Can you expend this to talk about this motivation?